tests/jerry/es2015/regression-test-issue-2733.js

```diff
@@ -41,7 +41,12 @@ function fz_collect_values(value) {
     for (var i = 0; i < prop_names.length; i++) {
         var prop_name = prop_names[i]
         if (!fz_starts_with(prop_name, "fz_")) {
-            fz_collect_values(value[prop_name])
+            try {
```

Comment:
It is not so good we have to touch a regression test for this new feature. How does this effect the original issue (#2733)?

Reply:
After a bit of investigation I realized that the original test case was invalid, and the only reason why it worked was the "luckiness" of the property enumeration order. Therefore I deleted this file.